src/utils/errors/errorMapping.json

```
@@ -0,0 +1,81 @@
+    "New size of oracle buffer should be positive": "New size of oracle buffer should be positive",
+    "OLD": "Internal error: Rate oracle not matured enough",
+    "x must be > 0": "Internal error: the value must be positive in BitMath",
+    "SafeMath: addition overflow": "Internal error: addition overflow",
+    "SafeMath: subtraction overflow": "ERC20: transfer amount exceeds balance",
+    "SafeMath: multiplication overflow": "Internal error: multiplication overflow",
+    "ERC20: transfer from the zero address": "Internal error: ERC20: transfer from the zero address",
+    "ERC20: transfer to the zero address": "Internal error: ERC20: transfer to the zero address",
+    "ERC20: transfer amount exceeds balance": "ERC20: transfer amount exceeds balance",
+    "ERC20: mint to the zero address": "ERC20: mint to the zero address",
+    "ERC20: burn from the zero address": "ERC20: burn from the zero address",
+    "ERC20: burn amount exceeds balance": "ERC20: burn amount exceeds balance",
+    "ERC20: approve from the zero address": "ERC20: approve from the zero address",
+    "ERC20: approve to the zero address": "ERC20: approve to the zero address",
+    "CT_CALLER_MUST_BE_LENDING_POOL": "Internal error: Caller must lending pool",
+    "CT_INVALID_MINT_AMOUNT": "Internal error: Invalid aToken amount to mint",
+    "CT_INVALID_BURN_AMOUNT": "Internal error: Invalid aToken amount to burn",
+    "Division by zero": "Internal error: Division by zero in aToken",
+    "overflow": "Internal error: Overflow in aToken",
+    "overflow in toUint160": "Internal error: Overflow when casting to Uint160",
+    "overflow in toInt128": "Internal error: Overflow when casting to Int128",
+    "overflow in toInt256": "Internal error: Overflow when casting to Int256",
+    "denominator underflows": "Internal error: Denominator underflows in SqrtPriceMath",
+    "starting px must be > quotient": "Internal error: Next price should be higher than current price in SqrtPriceMath",
+    "starting price must be > 0": "Internal error: Starting price not initialized in SqrtPriceMath",
+    "liquidity must be > 0": "Internal error: Liquidity must be positive in tick range",
+    "tick outside of range": "Internal error: Tick outside of range in TickMath",
+    "price outside of range": "Internal error: Price outside of range in TickMath",
+    "Wad Ray Math: 49": "Internal error: addition overflow in WadRayMath",
+    "Wad Ray Math: 50": "Internal error: division by zero in WadRayMath",
+    "PositionNetZero": "No need to unwind a net zero position",
+    "CannotLiquidate": "Position is not liquidatable",
+    "OnlyOwnerCanUpdatePosition": "No approval to update this position",
+    "OnlyVAMM": "No approval for this operation",
+    "OnlyFCM": "No approval for this operation",
+    "CannotSettleBeforeMaturity": "Cannot settle before maturity",
+    "NotEnoughFunds": "No enough funds to perform this operation",
```

```suggestion
    "NotEnoughFunds": "Not enough funds to perform this operation",
```

can you double check all?

---

https://github.com/Voltz-Protocol/v1-sdk/pull/132/commits/a2e1124cf0eb84af5b18f0253497806f52030238